src/pks/transport/transport_ats_pk.cc

```diff
@@ -52,6 +53,17 @@ Transport_ATS::Transport_ATS(Teuchos::ParameterList& pk_tree,
   PK(pk_tree, glist, S, solution),
   PK_PhysicalExplicit<Epetra_Vector>(pk_tree, glist, S, solution)
 {
+
+
+  key_ = Keys::readKey(*plist_, domain_, "primary variable", "total_component_concentration");
+  tcc_key_ = key_;
+  
+  // set up the primary variable solution, and its evaluator
+  Teuchos::ParameterList& FElist = S->FEList();
+  Teuchos::ParameterList& pv_sublist = FElist.sublist(key_);
+  pv_sublist.set("evaluator name", key_);
```

Comment:
"evaluator name" is no longer used for anything.  So you don't need to set this.  The name is assumed to be the ParameterList::name() value.